<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import</class>
 <widget class="QDialog" name="Import">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 134), stop:0.965174 rgba(255, 255, 255, 255));
font: 75 italic 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>导入表 解析</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>561</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>291</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 75 italic 10pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 10pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 134), stop:0.965174 rgba(255, 255, 255, 255));
font: 75 italic 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>返回主界面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Import</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
